content/admin/releases/7.10.x/_index.en.adoc

```diff
@@ -9,6 +9,205 @@ weight: 9890
+* PR: https://github.com/salesagility/SuiteCRM/pull/7844[7844^] - SuiteP: Add html data tags to allow module and field identification
+* PR: https://github.com/salesagility/SuiteCRM/pull/7837[7837^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7836[7836^] - Robo task to compile css in a custom theme
+* PR: https://github.com/salesagility/SuiteCRM/pull/7834[7834^] - Workflow: Properly delete records which are marked as deleted
+
+[discrete]
+
+==== pass:[<i class="fa fa-bug fa-1x"></i>] Bug Fixes
+
+* PR: https://github.com/salesagility/SuiteCRM/pull/7659[7659^] - Icons not rendering properly in Alerts
+* PR: https://github.com/salesagility/SuiteCRM/pull/7655[7655^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7648[7648^] - Case Module: Description field not showing after Save and continue
+* PR: https://github.com/salesagility/SuiteCRM/pull/7650[7650^] - 'customMetadate' typo in DashletGeneric.php
+* PR: https://github.com/salesagility/SuiteCRM/pull/7643[7643^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7622[7622^] - Make the code:coverage Robo command work outside of CI
+* PR: https://github.com/salesagility/SuiteCRM/pull/7641[7641^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7396[7396^] - Update button clears DateTime parameter in Reports Module
+* PR: https://github.com/salesagility/SuiteCRM/pull/7638[7638^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7315[7315^] - Adding parameter date field in Reports module causes error in Browser console
+* PR: https://github.com/salesagility/SuiteCRM/pull/7627[7627^] - Update sugar_3.js to fix a MassUpdate undefined error
+* PR: https://github.com/salesagility/SuiteCRM/pull/7587[7587^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7586[7586^] - Unnecessary include in UserService
+* PR: https://github.com/salesagility/SuiteCRM/pull/7529[7529^] - Codacy
+* PR: https://github.com/salesagility/SuiteCRM/pull/7525[7525^] - API Create Relationship via Link
+* PR: https://github.com/salesagility/SuiteCRM/pull/7515[7515^] - Scheduled Reports: Fix report name relation and popup search
+* PR: https://github.com/salesagility/SuiteCRM/pull/7428[7428^] - Issue: https://github.com/salesagility/SuiteCRM/issues/7427[7427^] - Show logs lines that was made by anonymous
+* PR: https://github.com/salesagility/SuiteCRM/pull/7195[7195^] - Inspections compatibility
+* PR: https://github.com/salesagility/SuiteCRM/pull/7193[7193^] - Remove Unused Import
+* PR: https://github.com/salesagility/SuiteCRM/pull/7141[7141^] - Type casting
+* PR: https://github.com/salesagility/SuiteCRM/pull/6765[6765^] - Issue: https://github.com/salesagility/SuiteCRM/issues/321[321^] - Hitting enter in the password input saves the user but not the password
+* PR: https://github.com/salesagility/SuiteCRM/pull/6750[6750^] - Issue: https://github.com/salesagility/SuiteCRM/issues/4754[4754^] - Remove PHP4 style constructors
```

Comment:
This should be in the Development section.

Reply:
Sorted